test/unit/adapters/http.js

```diff
@@ -1,11 +1,16 @@
 var axios = require('../../../index');
 var http = require('http');
+var https = require('https');
 var net = require('net');
 var url = require('url');
 var zlib = require('zlib');
 var assert = require('assert');
 var fs = require('fs');
-var server, proxy;
+var path = require('path');
+require('ssl-root-cas')
+  .inject()
```

Comment:
this method is deprecated in this dependency's documentation https://git.coolaj86.com/coolaj86/ssl-root-cas.js#inject, can we use something else?

Reply:
Good catch, `ssl-root-cas` module is not used anymore anyway